package.json

```diff
@@ -19,50 +19,55 @@
     "start": "ember serve",
     "test": "ember test",
     "test:all": "ember try:each",
-    "test:node": "for i in node-tests/*/*; do mocha $i; done"
+    "test:node": "( for i in node-tests/*/*; do mocha $i || exit $?; done )"
```

Comment:
Exit code of a for loop is the exit code of last iteration. So that's not very helpful here. To avoid wrong test result we use a subshell that is exit as soon as one test fails. Have found that trick here: https://stackoverflow.com/questions/14059342/how-to-get-the-exit-status-a-loop-in-bash

This will stop the tests as soon as the first file fails. It will not process later tests.

Reply:
Good catch!